.github/workflows/POST_RELEASE.yml

```diff
@@ -64,8 +64,3 @@ jobs:
         BPMN_IO_EMAIL: ${{ secrets.BPMN_IO_EMAIL }}
         BPMN_IO_USERNAME: ${{ secrets.BPMN_IO_USERNAME }}
       run: tasks/stages/update-website
-  update-translations:
```

Comment:
Is there a reason you moved this?

Reply:
I just moved it back because there was really no reason to do this.